src/utils/__tests__/sortPsp.test.ts

```diff
@@ -0,0 +1,57 @@
+import { sortBy } from "./../../utils/SortUtil";
+
+const bundles = [
+  { taxPayerFee: 100 },
+  { taxPayerFee: 200 },
+  { taxPayerFee: 150 },
+  { taxPayerFee: undefined },
+  { taxPayerFee: 0 },
+];
+
+test("sortBy function sorts bundles correctly in ascending order", () => {
+  const sorted = JSON.parse(JSON.stringify(bundles)).sort(
+    sortBy("taxPayerFee", "asc")
+  ); // Deep copy
+  expect(sorted).toEqual([
+    { taxPayerFee: 0 }, // 0 should come first
+    { taxPayerFee: 100 },
+    { taxPayerFee: 150 },
+    { taxPayerFee: 200 },
+    { taxPayerFee: undefined },
+  ]);
+});
+
+test("sortBy function places undefined values at the end in ascending order", () => {
+  const sorted = JSON.parse(JSON.stringify(bundles)).sort(
+    sortBy("taxPayerFee", "asc")
+  ); // Deep copy
+  expect(sorted[sorted.length - 1].taxPayerFee).toBeUndefined();
+});
+
+test("sortBy function sorts bundles correctly in descending order", () => {
```

Comment:
add tests to check right sorting also for literal fields too (the case bundles are sorted by name

Reply:
literal sorting is not affected by this PR, should we test it anyway?

Reply:
Why is not affected? Both sorting functionality are performed by the same method that have been touched here + better have an extra test than nothing

Reply:
yes but the function only changed from this
`return fieldA && fieldB ? (fieldA > fieldB ? order : -order) : -order;`
to basically this
`fieldA !== undefined && fieldB !== undefined ? (fieldA > fieldB ? order : -order) : -order;`

this allows us to sort correctly value = 0 (the 0 was buggy since fieldA && fieldB fails when is 0 && 0) 

the literal sorting would never fail the fieldA && fieldB test since empty string for the name pspBusinessName can never be empty string. this is why i say it's not impacted (infact, the bug on the pspBusinessName never happened in the first place)

Reply:
Yes I know, but theoretically speaking the code was touched and for a better coverage a test with string ordering will only add a guarantees about both ordering methods.
Moreover this will add a check for future development making test fail in case of any introduced regression. 
I don't see any problem adding an additional test, do you?